services/libs/integrations/package.json

```diff
@@ -32,6 +32,7 @@
     "@crowd/common": "file:../common",
     "@crowd/logging": "file:../logging",
     "@crowd/types": "file:../types",
+    "@crowd/redis": "file:../redis",
```

Comment:
`@crowd/redis` is surprisingly a big library - I would suggest doing what I did in `@crowd/integrations` library where I put `ICache` interface in types and used `RedisCache` class object instance through it without the need to add the dependency on `@crowd/redis` here.